etl/Makefile

```diff
@@ -0,0 +1,12 @@
+.PHONY: all clean
+
+clean_county_populations.json: raw_county_populations.json
+	python3 get_county_populations.py $< $@
```

Comment:
TIL, didn't know Python could do STDOUT, but makes sense because of how utilitarian the language is (at least I perceive it to be utilitarian). Changed here: d034514